x-pack/test/functional/apps/ml/aiops/explain_log_rate_spikes.ts

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FtrProviderContext } from '../../../ftr_provider_context';
+import { TestData } from './types';
+import { farequoteDataViewTestData } from '../data_visualizer/index_test_data';
+
+export default function ({ getPageObject, getService }: FtrProviderContext) {
+  const headerPage = getPageObject('header');
+  const esArchiver = getService('esArchiver');
+  const ml = getService('ml');
+
+  function runTests(testData: TestData) {
+    it(`${testData.suiteTitle} loads the source data in explain log rate spikes`, async () => {
+      await ml.testExecution.logTestStep(
+        `${testData.suiteTitle} loads the saved search selection page`
+      );
+      await ml.aiops.navigateToIndexPatternSelection();
+
+      await ml.testExecution.logTestStep(
+        `${testData.suiteTitle} loads the explain log rate spikes page`
+      );
+      await ml.jobSourceSelection.selectSourceForExplainLogRateSpikes(
+        testData.sourceIndexOrSavedSearch
+      );
+    });
+
+    it(`${testData.suiteTitle} displays index details`, async () => {
+      await ml.testExecution.logTestStep(`${testData.suiteTitle} displays the time range step`);
+      await ml.aiops.assertTimeRangeSelectorSectionExists();
+
+      await ml.testExecution.logTestStep(`${testData.suiteTitle} loads data for full time range`);
+      await ml.aiops.clickUseFullDataButton(testData.expected.totalDocCountFormatted);
+      await headerPage.waitUntilLoadingHasFinished();
+
+      await ml.testExecution.logTestStep(
+        `${testData.suiteTitle} displays elements in the doc count panel correctly`
+      );
+      await ml.aiops.assertTotalDocCountHeaderExist();
+      await ml.aiops.assertTotalDocCountChartExist();
+
+      await ml.testExecution.logTestStep(
+        `${testData.suiteTitle} displays elements in the page correctly`
+      );
+
+      await ml.aiops.assertSearchPanelExist();
+
+      await ml.testExecution.logTestStep('displays empty prompt');
+      await ml.aiops.assertNoWindowParametersEmptyPromptExist();
+    });
+  }
+
+  describe('index based', function () {
```

Comment:
This test suite name should be updated

Reply:
Fixed in [5b14409](https://github.com/elastic/kibana/pull/138387/commits/5b1440910fdebf4a0178b90a1aa7dc9cbbf179c2).